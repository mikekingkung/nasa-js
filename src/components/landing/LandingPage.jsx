import landingPageImage from  '../../images/landingPageImage.jpg';

const LandingPage = () => {
    return (
        <img
            className="img-fluid"
            src={landingPageImage}
            alt="Learn Space Ltd"
        />
    );
};

export default LandingPage;
