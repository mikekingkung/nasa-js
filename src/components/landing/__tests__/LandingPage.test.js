import React from 'react';
import { render} from '@testing-library/react';
import LandingPage from '../LandingPage';

describe('<Landing Page component', () => {
    it('Correctly renders' ,() => {
        const wrapper = render(<LandingPage/>);
        expect(wrapper).toMatchSnapshot();
    });
});