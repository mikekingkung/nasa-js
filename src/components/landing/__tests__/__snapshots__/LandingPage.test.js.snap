// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Landing Page component Correctly renders 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div>
        <p
          data-testid="welcome"
        >
          Welcome to the Learn Space application. The application makes use of the NASA Video and Image API. The application allows keyword searches. Results are limited to 10 currently per search. It is also possible to save links and review them on the user details page.
        </p>
        <img
          alt="Learn Space Ltd"
          class="img-fluid"
          src="landingPageImage.jpg"
        />
      </div>
    </div>
  </body>,
  "container": <div>
    <div>
      <p
        data-testid="welcome"
      >
        Welcome to the Learn Space application. The application makes use of the NASA Video and Image API. The application allows keyword searches. Results are limited to 10 currently per search. It is also possible to save links and review them on the user details page.
      </p>
      <img
        alt="Learn Space Ltd"
        class="img-fluid"
        src="landingPageImage.jpg"
      />
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
